src/material-experimental/mdc-form-field/form-field.ts

```diff
@@ -558,10 +556,17 @@ export class MatFormField
     if (!this._hasOutline() || !this._floatingLabel) {
       return;
     }
+    if (!this._labelScale) {
+      this._labelScale = Number(
+        getComputedStyle(this._elementRef.nativeElement).getPropertyValue(
+          '--mat-mdc-form-field-floating-label-scale',
```

Comment:
Could we instead make this dynamic and use the variable in a CSS computation, or is this expected to be changing rarely, in which case a comment would also be nice to just state that constraint.

Reply:
Done